kong.conf.default

```diff
@@ -1334,3 +1334,21 @@
                                  # server.
                                  #
                                  # See https://github.com/openresty/lua-nginx-module#lua_socket_pool_size
+
+#untrusted_lua = on              # When enabled, Kong will accept and run
+                                 # untrusted lua code on plugins and features
+                                 # that accept it.
+
+#untrusted_lua_sandbox = on      # When enabled, Kong uses a secure sandbox
+                                 # for running untrusted lua code. If disabled
+                                 # untrusted lua code has unrestricted access
+                                 # to the global context
+
+#untrusted_lua_sandbox_requires =
```

Comment:
"modules" instead of "requires"?

Reply:
it's not the same. This makes some requires available to a fake `require` function. So, this just allows a list of things that can be "required", thus `requires`.

`modules` can be confused with what `untrusted_lua_sandbox_environment` provides.

Reply:
imo this should stay as close to standard Lua as possible, that we limit the modules to a specific subset is fine, but they are still modules. So using "requires" is just confusing to the user.